docker/app_img/Dockerfile

```diff
@@ -0,0 +1,64 @@
+ARG UBUNTU_BASE="ubuntu:22.04"
+
+FROM ${UBUNTU_BASE} AS image_build
+
+ARG OTACLIENT_WHL
+COPY ./${OTACLIENT_WHL} /tmp
+
+ARG OTACLIENT_INSTALLATION="/otaclient"
+ENV OTACLIENT_VENV=${OTACLIENT_INSTALLATION}/venv
+
+RUN set -eux; \
+    apt-get update -qq; \
+    apt-get install --no-install-recommends -y \
+        python3-venv \
+        python3-pip; \
+    # setup virtual env
+    mkdir -p ${OTACLIENT_VENV}; \
+    cd ${OTACLIENT_INSTALLATION}; \
+    python3 -m venv ${OTACLIENT_VENV}; \
+    . ${OTACLIENT_VENV}/bin/activate; \
+    # install otaclient
+    python3 -m pip install -U pip; \
+    python3 -m pip install /tmp/${OTACLIENT_WHL}; \
+    # cleanup
+    apt-get clean; \
+    \
+    # NOTE: copy from docker official python3 dockerfile
+	find ${OTACLIENT_VENV} -depth \
+		\( \
+			\( -type d -a \( -name test -o -name tests -o -name idle_test -o -name __pycache__ \) \) \
+			-o \( -type f -a \( -name '*.pyc' -o -name '*.pyo' -o -name 'libpython*.a' \) \) \
+		\) -exec rm -rf '{}' + \
+	; \
+	\
+    rm -rf \
+        /root/.cache \
+        /tmp/* \
+        /var/lib/apt/lists/* \
+        /var/tmp/*
+
+FROM ${UBUNTU_BASE}
```

Comment:
Why we need multiple stage?(To download python on venv?)
I wonder, if we need to use venv on the docker.

Reply:
In the first stage, the virtual environment is boostrapped with `python3-pip` and `python3-ven` package. 
In the second stage, we only need `python3` package, then the otaclient in the virtual environment can run as expected.

Using virtual env is for making otaclient installation portable and controllable.